DocsAndHospitals/Services/HospitalService.cs

```diff
@@ -0,0 +1,208 @@
+﻿using DocsAndHospitals.Models;
+using System;
+namespace DocsAndHospitals.Services;
+
+public class HospitalService
+{
+    public void DisplayHospitals(Hospital[] hospitals)
+    {
+        foreach (var h in hospitals)
+            Console.WriteLine($"ID: {h.KNumber}, Name: {h.Name}, Address: {h.Address}, Phone: {h.PhoneNumber}");
```

Comment:
DI violation. Rework and use abstraction for it - you cannot be dependent on Console in Business Logic service